python/hail/expr/types.py

```diff
@@ -555,12 +555,12 @@ def __init__(self, name, typ):
 
 class TStruct(Type):
     """
-    Hail type corresponding to :class:`hail.representation.Struct`.
+    Hail type corresponding to :class:`hail.utils.Struct`.
 
-    .. include:: hailType.rst
+    .. include:: ../_templates/hailType.rst
 
     - `expression language documentation <types.html#struct>`__
-    - in Python, values are instances of :class:`hail.representation.Struct`
+    - in Python, values are instances of :class:`hail.genetics.Struct`
```

Comment:
There is a Struct in genetics?  How's that different than the Struct in utils?

Reply:
bug.